fars/booking/forms.py

```diff
@@ -123,17 +151,20 @@ def save_repeating_booking_group(self, booking):
         group.save()
         booking.repeatgroup = group
         skipped_bookings = []
+        frequency = data.get('frequency')
+        repeat_until = data.get('repeat_until')
+        is_repetition = False
 
         # Copy booking for every repetition
-        while(booking.start.date() <= data.get('repeat_until')):
+        while(booking.start.date() <= repeat_until):
             overlapping = booking.get_overlapping_bookings()
             if overlapping:
                 skipped_bookings.append(str(booking))
             else:
-                booking.save()
+                booking.save(skip_gcal=is_repetition)
```

Comment:
I suggest all the logic for whether to skip gcal should all be in the same place (logically the model in this case). This way you don't have to go searching for it when you update the code.